core/ts/Poll.ts

```diff
@@ -857,9 +903,10 @@ class Poll implements IPoll {
   };
 
   /**
-   * Tally a batch of Ballots and update this.results
+   * This method tallies a ballots and updates the tally results.
+   * @returns the circuit inputs for the TallyVotes circuit.
    */
-  public tallyVotes = () => {
+  public tallyVotes = (): unknown => {
```

Comment:
Why unknown here, where for instance on processMessages you added a specific type?